src/app/zwiazek/kalendarz/_components/Dates/Dates.tsx

```diff
@@ -0,0 +1,4 @@
+export const addMonth = (date: Date, offset = 1) =>
```

Comment:
Maybe having a default value here is not such a good idea after all 😅 
Let's change it back to `: number`.

Also how does it handle `NaN` and `non-integer values`? 
A test for that would be nice.